purchase_requisition_auto_rfq/view/purchase_requisition.xml

```diff
@@ -11,10 +11,16 @@
                 <button
                     name="auto_rfq_from_suppliers"
                     type="object"
-                    string="Generate quotations from suppliers"
+                    string="Generate RFQs for suppliers"
                     attrs="{'invisible': ['|', ('line_ids', '=', []), ('state', '!=', 'in_progress')]}"
                 />
             </xpath>
+            <xpath expr="//field[@name='line_ids']/tree" position="inside">
+                <field name="prod_has_supplier" invisible="1" />
+            </xpath>
+            <xpath expr="//field[@name='line_ids']/tree" position="attributes">
+                <attribute name="decoration-danger">not prod_has_supplier</attribute>
+            </xpath>
```

Comment:
Maybe we don't need to close this PR. This feature is still good here IMHO, isn't it?

Reply:
I have modified this pr and now it only has the improvement mentioned in the description..